index.html

```diff
@@ -8,6 +8,96 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+   <div id="home-page"> 
+        <div id="top-info">
+            <div id="top-left-info">
+                <div class="left-info">Travel</div>
+                <div class="left-info">Goo</div>
+            </div>
+            <div id="top-mid-info">
+               <div class="mid-info">Home</div>
+               <div class="mid-info">About Us</div>
+               <div class="mid-info">Packages</div>
+            </div>
+            <div id="top-right-info">
+                <img class="right-info" src="./images/star.svg">
+                <img id="circle" src="./images/profile.svg">
+            </div> 
+        </div> 
+        <div id="centralised-info">
+            <div id="mid-quote">
+                <div class="quote">Your Imagination Is</div>
+                <div id="line1" class="quote">Your Only limit</div>
+            </div>
+            <div id="our-role">
+                <div>We always try to make our customer Happy, We provide all kind of facilities</div>
+                <div id="line2">Your Satisfaction is our main priority</div>
+            </div>
+            <div id="button">
+                <div id="green-box">Discover more</div>
+            </div>
+        </div>
+    </div> 
+    <div id="second-page">
+        <div id="title">
+            <div id="edit">Our</div>
+            <div>Service</div>
+        </div>
+        <div id="content01">
+            <div class="info">
+                <img class="img-sizingx" src="./images/ticketBooking.svg">
+                <div class="heading">Ticket Booking</div>
+                <div class="t1">We book all kind of national or international</div>
+                <div class="t1">ticket for your destination.</div>
+            </div>
+            <div id="info-mid" class="info">
+                <img class="img-sizing" src="./images/hotelBooking.svg">
+                <div class="heading">Hotel Booking</div>
+                <div class="t1">You can easily book according to your</div>
+                <div class="t1">budjet hotel by our website.</div>
+            </div>
+            <div class="info">
+                <img class="img-sizing" src="./images/tourPlan.svg">
+                <div class="heading">Tour Plan</div>
+                <div class="t1">We provide you the best plan within a short</div>
+                <div class="t1">time explore more.</div>
+            </div>
+        </div>      
+    </div>  
+    <div id="content02">
+        <div id="best-service">
+            <div  class="bold">We always try to give you</div>
```

Comment:
I told before, you should not use properties like 'bold' 'sizing' in your classnames or id. Right now , we are implementing the normal css approach.. we are not implementing the utility classes approach. 

So use b tag to make the text bold

Reply:
yes i changed them